### PR TITLE
:sparkles: Run yip directly for stages

### PR DESCRIPTION
**Pull request by Itxaka**
Runs yip directly instead of using elemental and drop elemental from initramfs

Fixes: https://github.com/kairos-io/kairos/issues/1069

### CODE REVIEW

File: internal/utils/cloudinit.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+			plugins.Sysctl,
+			plugins.User,
+			plugins.SSH,
+			plugins.LoadModules,
+			plugins.Timesyncd,
+			plugins.Systemctl,
+			plugins.Environment,
+			plugins.SystemdFirstboot,
+			plugins.DataSources,
+			plugins.Layout,
```
Comment (mudler) [inline internal/utils/cloudinit.go:47]: wondering about the layout plugin here, shall we just have it in `yip`?

**Review by mudler — APPROVED** [review]
Looking good! that's actually great!

File: internal/utils/cloudinit.go (hunk 1 above)
Comment (Itxaka) [reply internal/utils/cloudinit.go:47]: It comes from yip directly, and should expand the partitions and all that crap. Not sure why elemental did have the plugin separated, if it was to inject the fs and runners and such or what. Will have a look and potentially send patches to yip, but as far as I can see, the bundled plugin in yip should expand the partitions correctly

Comment (mudler) [reply internal/utils/cloudinit.go:47]: IIRC it was due to sgdisk/parted, no? IIRC the yip standard plugin supported only BIOS, not GPT disks

Comment (mudler) [reply internal/utils/cloudinit.go:47]: yep, it used indeed the elemental-cli partitioner instead: https://github.com/kairos-io/elemental-cli/blob/04802f037647560f4d8a4a4f382ec1cb375f9d72/pkg/cloudinit/layout_plugin.go#L36

Comment (mudler) [reply internal/utils/cloudinit.go:47]: also.... I remember patching some parts too in our fork

Comment (Itxaka) [reply internal/utils/cloudinit.go:47]: argh, that sucks. Either we bundle a custom version here or we sent the patches to yip

**Pull request merged**